### ISSUE TITLE
set cutoffs

### ISSUE THREAD

**Issue by dzaccook** [dfguan/purge_dups#123]
Hi,
I have a ~20Gb assembled genome and the estimated size was 11Gb. The sequencing depth is about 50x (PB CCS reads). The automatically cutoffs is 5 17 27 33 55 99. 
[M::calcuts] Find 1 peaks
[M::calcuts] mean: 26, peak: 22, mean larger than peak, treat as diploid assembly
[W::calcuts] mean is not significantly different with peak, please recheck the cutoffs.
Using the automatically cutoffs, I got a 12 Gb purged genome and BUSCO is C:98.9%[S:87.5%,D:11.4%],F:0.2%,M:0.9%,n:1614. The ratio of duplicated BUSCOs and genome size are bigger with  this cutoffs.
Could you give some suggestions to set the cutoffs manually? Thank you!
![PB cov](https://user-images.githubusercontent.com/23490134/193183262-f2b2f72c-9486-430f-b72e-2b521d67a47b.png)

**Comment by dfguan** [comment]
Hello,

Sorry for the late reply. Let's suppose you have an expected read depth of
D, you may change 33 in the cutoff file to 3/4*D. You can also try to set a
lower higher depth for purge_dups, this will remove more repetitive
sequences, although you may risk losing some real repeat sequence.

Hope this helps. Best, Dengfeng.

**Comment by dzaccook** [comment]
Thanks a lot.

**Comment by NiccoloBassetti** [comment]
Hi @dfguan, 

I have a few questions related to dzaccook`s ones, so I added them here to this issue. 

1. Am I understading correctlt that the rational behind manual setup it to have the cutoffs "cutting" the distribution in the points where it changes slopes?

2. For example, here below a coverage plot from my assembly.
As I also get the message that mean (15) is not different from peak (10), I should adjust the cutoff and it should be somewhere between 10 and 15?

3. You suggested 3/4 *D, I assume D is the sequencing depth (total reads length/genome size)?

![example](https://user-images.githubusercontent.com/98760002/204912137-918d7725-266f-470e-8757-4c10b6726c62.png)

Here is some background on my data. 
I am working on a genome of an heterozygous plant, estimated size of ~500 Mb.
I have ~10 Gb Nanopore .fq reads, so a coverage of just ~20x.
So far my pipeline is: Flye (with --keep haplotypes ,mode) --> Medaka --> purge_dups --> HapDup. 
After Flye/medaka I got a size of ~600 Mb, so 1/5 bigger that estimated genome size. 

4. An assembly bigger than the genome size does it explain the right shoulder to the right of the peak in the cov plot, between 15x and 20x? 

Cheers!

PS. Thanks for developing purge_dups and your availability to answer questions. This github page is full of resources and good discussions!
